### PR TITLE
[port] update prevector.h

### PR DESCRIPTION
**Pull request by sickpig**
`prevector.h`:

```c++
/** Implements a drop-in replacement for std::vector<T> which stores up to N
 *  elements directly (without heap allocation). The types Size and Diff are
 *  used to store element counts, and can be any unsigned + signed type.
 *  ...
 **/
```

This is mainly due to bring to the project the perf gain and fixed that this subsys received by the means of this series of Core PRs:

bitcoin/bitcoin#12324
bitcoin/bitcoin#14715
bitcoin/bitcoin#14651
bitcoin/bitcoin#14266
bitcoin/bitcoin#14028
bitcoin/bitcoin#14030
bitcoin/bitcoin#12549
bitcoin/bitcoin#9856
bitcoin/bitcoin#11151

### CODE REVIEW

**Review by ptschip — APPROVED** [review]
Looks like a useful set of ports with a good perf improvement.

File: .travis.yml [hunk 1]
```diff
@@ -146,11 +146,10 @@ before_script:
     - set -o errexit; source .travis/before_script.sh
 script:
     - export CONTINUE=1
+<<<<<<< HEAD
```
Comment (gandrewstone) [inline .travis.yml]: Merge error!

Comment (sickpig) [reply .travis.yml]: Ouch, Will fix way to many rebases and interleaved PRs

Comment (sickpig) [reply .travis.yml]: fixed

**Comment by sickpig** [comment]
rebased

**Pull request merged**